.NET/Microsoft.Recognizers.Text.DateTime/German/Parsers/GermanDateParserConfiguration.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+
+using Microsoft.Recognizers.Text.DateTime.Utilities;
+using Microsoft.Recognizers.Definitions.German;
+using Microsoft.Recognizers.Text.Number;
+
+namespace Microsoft.Recognizers.Text.DateTime.German
+{
+    public class GermanDateParserConfiguration : BaseOptionsConfiguration, IDateParserConfiguration
+    {
+        public string DateTokenPrefix { get; }
+
+        public IExtractor IntegerExtractor { get; }
+
+        public IExtractor OrdinalExtractor { get; }
+
+        public IExtractor CardinalExtractor { get; }
+
+        public IParser NumberParser { get; }
+
+        public IDateTimeExtractor DurationExtractor { get; }
+
+        public IDateTimeParser DurationParser { get; }
+
+        public IEnumerable<Regex> DateRegexes { get; }
+
+        public IImmutableDictionary<string, string> UnitMap { get; }
+
+        public Regex OnRegex { get; }
+
+        public Regex SpecialDayRegex { get; }
+
+        public Regex NextRegex { get; }
+
+        public Regex ThisRegex { get; }
+
+        public Regex LastRegex { get; }
+
+        public Regex UnitRegex { get; }
+
+        public Regex WeekDayRegex { get; }
+
+        public Regex MonthRegex { get; }
+
+        public Regex WeekDayOfMonthRegex { get; }
+
+        public Regex ForTheRegex { get; }
+
+        public Regex WeekDayAndDayOfMothRegex { get; }
+
+        public Regex RelativeMonthRegex { get; }
+
+        //The following three regexes only used in this configuration
+        //They are not used in the base parser, therefore they are not extracted
+        //If the spanish date parser need the same regexes, they should be extracted
```

Comment:
spanish should be German, right? :)
Did you use the English class as base? Or the Spanish ones?
The ideal case is to use the English ones.

Reply:
Yes, i used the English class as base, this comment is from there. I left it there, because, as you suggested, I started working on the numbers and did not change anything in DateTime or NumbersWithUnits apart from what was necessary to build the project and provide the basic structure based on the English one.

Reply:
OK